shell/shell.js

```diff
@@ -16,6 +16,177 @@
 
 const { classes: Cc, interfaces: Ci, results: Cr, utils: Cu } = Components;
 const { Runtime } = Cu.import('resource://qbrt/modules/Runtime.jsm', {});
+const { Services } = Cu.import('resource://gre/modules/Services.jsm', {});
+
+function KeyAccelerators(OS, keyModifierAliases) {
```

Comment:
Yes, I imagine that other apps would want to use them as well, not only the shell. I'm unsure how best to expose them, however. The easy answer is to put them into a JSM, modules/KeyAccelerators.jsm. And that's probably the best approach for now. In the future I'd rather use an ES6 or CommonJS loader. But that's a separable concern, so we can do that later.